packages/client/components/TaskEditor/withEmojis.tsx

```diff
@@ -76,16 +76,20 @@ const withEmojis = (ComposedComponent) => {
     renderModal = () => {
       const {query} = this.state
       const {editorState} = this.props
-      this.cachedCoords = getDraftCoords() || this.cachedCoords
+      this.cachedCoords = getDraftCoords()
       return (
-        <EmojiMenuContainer
-          removeModal={this.removeModal}
-          menuItemClickFactory={this.menuItemClickFactory}
-          query={query}
-          menuRef={this.menuRef}
-          editorState={editorState}
-          originCoords={this.cachedCoords}
-        />
+        <>
+          {this.cachedCoords && (
```

Comment:
-2 this just disables the menu for this situation, which is worse than showing it in the wrong place

Reply:
Well the menu is not entirely disabled, we are only preventing it from rendering with `undefined` coordinates,  its actually not so long when we receive actual `coordinate`  values from `getVisibleSelectionRect` instead of `undefined`.